### DISCUSSION TITLE
[SOLVED] How to enter Configuration?

### DISCUSSION THREAD

**Discussion by 0xlne** [2bllw8/anemo#36] [Q&A]
The F-Droid screenshots show a configuration screen, but this screen cannot be found after installation, neither in the hamburger menu nor in the 3-button menu.
Tested on the following devices:
Xiaomi Mix Fold 2 (Android 12) & Redmagic 6 (Android 12), Samsung Galaxy Note 10+ (Android 11).
Anemo version: 2.5.0 F-Droid.

Would appreciate guidance how to reach this screen.

**Comment by Wjxfi** [comment]
https://user-images.githubusercontent.com/113500363/197365315-8d26556e-89bd-41ad-8869-bf5562d604cc.mp4
